dll/appcompat/apphelp/apphelp.c

```diff
@@ -26,7 +26,7 @@ const UNICODE_STRING InstalledSDBKeyName = RTL_CONSTANT_STRING(L"\\Registry\\Mac
 /* from dpfilter.h */
 #define DPFLTR_APPCOMPAT_ID 123
 
-#define MAX_GUID_STRING_LEN     sizeof("{12345678-1234-1234-0123-456789abcdef}")
+#define MAX_GUID_STRING_LEN   _countof("{12345678-1234-1234-0123-456789abcdef}") // == 39
```

Comment:
`error: implicit declaration of function '_countof'` // also, maybe C_ASSERT instead of comment?

Reply:
fixed